Modules/_sqlite/cursor.c

```diff
@@ -921,12 +921,12 @@ _sqlite3.Cursor.setinputsizes as pysqlite_cursor_setinputsizes
     sizes: object
     /
 
-Required by DB-API. Does nothing in pysqlite.
+Required by DB-API. Is a no-op in sqlite3.
```

Comment:
Any reason not to keep the "Does nothing" wording? I think "does nothing" is slightly better because "no-op" is jargon.

Reply:
OK, let's change it back to "does nothing" then. I don't think this is important enough to change everywhere, but it's better to stick to the more widely understood form.

Reply:
Sure, let's do that